### DISCUSSION TITLE
related backend setup

### DISCUSSION THREAD

**Discussion by shilpashingnapure** [Innovateninjas/Paws-Backend#7] [Q&A] [answered]
Hi @anirbanmajumder0  
facing some issue in setup
  step I did , some of steps are not mentioned in contribute.md , please let me know if I did correct or not 
   1. for .env firebase variable : 
       1. project overview -> service account -> generate new private key (with this i able to get all env variables which mentions in .env.example)
    2. for secret key -> 
        1. I have run this cmd to get secret key : 
               python manage.py shell -c "from django.core.management.utils import get_random_secret_key; print(get_random_secret_key())"
   3. db is also connected
   4. but when i try for migrate having this error 
![Screenshot 2024-05-12 at 6 56 32 PM (2)](https://github.com/Innovateninjas/Paws-Backend/assets/69167064/1907ac3c-4ba0-4299-af28-379dbfa85ace)

**Comment by Anirban-Majumder** [comment, marked as answer]
U havnt set up the db properly , first create a empty db then add the creds in the env , then migrate by using python manage.py makemigration and followed by python manage.py migrate

**Reply by shilpashingnapure** [reply]
Resolved , it happended becase of makemigrations is not detected for aniresfr
Thank you!!
